.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: ruby 
+rvm:
+  - 2.5.3
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database chestmasters_test;' -U postgres
```

Comment:
@learGitHub can you help change this to be the correct name for the test database?